### PR TITLE
[Core] Length for NurbsCurve and NurbsCurveOnSurface

### PR DESCRIPTION
**Pull request by tteschemacher**
Cheers,

**Description**
this PR adds the possibility to compute the length of nurbs curves and nurbs curve on surfaces and also in the brep configuration.

**Changelog**
- Integrate the length via integration points and determinant of the base vector
- computation of Jacobian and its Determinant

Thanks for the review!

### CODE REVIEW

File: kratos/tests/cpp_tests/geometries/test_nurbs_curve.cpp [hunk 1]
```diff
@@ -14,6 +14,7 @@
 #include <limits>
 
 // External includes
+#include <iomanip>
```
Comment (dagmawibekel) [inline kratos/tests/cpp_tests/geometries/test_nurbs_curve.cpp]: Do you need this include?

Comment (tteschemacher) [reply kratos/tests/cpp_tests/geometries/test_nurbs_curve.cpp]: thanks. it is deprecated, I forgot to remove it. it is gone

**Comment by tteschemacher** [comment]
@dagmawibekel @philbucher are we all set?

**Review by dagmawibekel — APPROVED** [review]

**Comment by tteschemacher** [comment]
Nice!

**Pull request merged**